server/index.js

```diff
@@ -1,18 +1,29 @@
 const express = require('express');
+
 const app = express();
 const PORT = 3000 || process.env.PORT;
 const bp = require('body-parser');
 const path = require('path');
+const mongoose = require('mongoose');
 
-console.log(__dirname + '../client/src/dist')
+mongoose.connect('mongodb://localhost/items');
+const itemModel = require('../database/mongooseModel.js');
 
-app.use(express.static(path.resolve(__dirname, '..', 'client', 'src','dist')));
+app.use(express.static(path.resolve(__dirname, '..', 'client', 'src', 'dist')));
 app.use(bp.json());
 
-app.get('/', function(req, res) {
-  res.send('Hello World!')
+app.get('/items/:id', (req, res) => {
+  itemModel.item.find(req.params, (err, data) => {
+    if (err) {
+      // eslint-disable-next-line no-console
+      console.log(err);
+    } else {
+      res.send(data[0]);
```

Comment:
I noticed you're using an index of data since the .find() method returns an array of objects. If you use the findOne() method, it returns a single object.

Reply:
Great idea! I'll switch that over for neater code.